acquia_lift.admin.unibar.inc

```diff
@@ -302,7 +80,7 @@ function acquia_lift_element_variation_details_form($form, &$form_state, $variat
   $form['agent'] = array(
     '#type' => 'hidden',
   );
-  // Variation number applicable to page variations.
+  // Variation number applicable to element variations.
```

Comment:
Looks like this is only used as a value and never populated by the client-side so there is no reason to have it here as a hidden form value.  Also no need to give it a comment when there is only one type of variation being created/edited.